pyodide-build/pyodide_build/install_xbuildenv.py

```diff
@@ -37,5 +38,5 @@ def main(args):
         ]
     )
     shutil.copytree(
-        xbuildenv_path / "site-packages", host_site_packages, dirs_exist_ok=True
+        xbuildenv_path / "site-packages-extras", host_site_packages, dirs_exist_ok=True
```

Comment:
Could you add a comment about what's happening here, and why it needs to be != from site-packages.

Reply:
The name change isn't technically necessary, it's intended as a clarification. The real `site_packages` directory is at
```
pyodide-root/packages/.artifacts/lib/python3.10/site-packages/
```
and is generated by
```sh
pip install -r requirements.txt -t $HOSTSITEPACKAGES
cp site-packages-extra $HOSTSITEPACKAGES
```
hence the name `site-packages-extra`.